docs/developers_guide/cfit_struct_c.md

```diff
@@ -0,0 +1,85 @@
+<!--
+(C) copyright 2020 VT SuperDARN, Virginia Polytechnic Institute & State University
+author: Kevin Sterne
+-->
+
+# Structure in C for the cFit compressed fitted ACF data
+
+## Summary
+
+A description of the data structure in the C language used to store the cFit Data.
+
+## Description
+
+### Introduction
+
+The cFit format is a compressed format for storing a limited sub-set of data from the output of the FitACF algorithm. Listed below is the C data structure <code>struct CFitData</code> used to store the cFit data.
+
+### Array Limits
+
+The array limits are defined as macros in the header <code>superdarn/limit.h</code>.
+```
+#define MAX_RANGE 300	
+```
+
+### `struct CFitData`
+
+The fitted ACF data structure is defined in the header <code>superdarn/cfitdata.h</code>.
+```
```

Comment:
hard to add the fix in suggestion but you can put the programming language next to the ``` to make highlighting to happen.

Reply:
Do you mean with the replacing of the <code> marks?  There were so many that at times I gave up on trying to get them all.  They seem to do the same job though, right?  I suppose I could go through and clean them a little better to make it more consistent.

Reply:
nah looking at the readthedocs it seems good.